tensilelite/Tensile/KernelWriter.py

```diff
@@ -162,7 +162,6 @@ class StateValues:
   lrvwUnrollB: int                       = 0
   lrvwUnrollMetadata: int                       = 0 # For Sparse Metadat
 
-  vgprValuDouble: bool                   = False
```

Comment:
vgprValuDouble was not set to true anywhere.
This was originally added by me in Tensile for DirectToLds.
However, this was necessary due to some scheduling bug and this was already removed in Tensile.
I removed it this time (to avoid adding code for unnecessary feature).